cfgov/unprocessed/js/molecules/Notification.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+// Required polyfills for <IE9.
+require( '../modules/polyfill/query-selector' );
+require( '../modules/polyfill/class-list' );
+
+// Required modules.
+var atomicCheckers = require( '../modules/util/atomic-checkers' );
+
+/**
+ * Notification
+ * @class
+ *
+ * @classdesc Initializes a new Notification molecule.
+ *
+ * @param {HTMLNode} element
+ *   The DOM element within which to search for the molecule.
+ * @returns {Object} An Notification instance.
+ */
+function Notification( element ) { // eslint-disable-line max-statements, inline-comments, max-len
+
+  var BASE_CLASS = 'm-notification';
+
+  // Bitwise flags for the state of this Expandable.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/bef83c3dd266a041482192d327447282b284b7f5 That comment was copied from expandables when it shouldn't have been. Said `Constants for the state of this Notification`